package/crds/cache.gcp.crossplane.io_cloudmemorystoreinstances.yaml

```diff
@@ -151,6 +154,9 @@ spec:
               atProvider:
                 description: CloudMemorystoreInstanceObservation is used to show the observed state of the CloudMemorystore resource on GCP.
                 properties:
+                  authString:
+                    description: 'AuthString: AUTH string set on the instance.'
+                    type: string
```

Comment:
This is likely why `check-diff` is failing -- it appears you haven't run `make generate` again and included changes 👍🏻

Reply:
Thanks for the reminder!